.transifex.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#
+# push the string automaticaly to transifex once the PR has been merged
+# https://docs.transifex.com/transifex-github-integrations/github-tx-client#integrating-the-client-with-travis-ci
+
+if [[ "$TRAVIS_BRANCH" != "master" ]]; then
+  echo "We are not on the master branch. We push translation only on master"
+  # analyze current branch and react accordingly
+  exit 0
+fi
+
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
+  echo "We are on a pull request. We push translation only after the PR merge"
+  # analyze current state PR merged or PR and react accordingly
+  exit 0
+fi
+
+echo "We push the translations to transifex"
+
+# set python3 and upgrade pip (default pip 10.0)
+pyenv global 3.7.1
+pip3 install --upgrade pip
+
+pip install virtualenv
+virtualenv ~/env
+source ~/env/bin/activate
+pip install transifex-client
+echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"api"$'\npassword = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc
+tx push -s
```

Comment:
With Python 3 pyenv/virtualenv shouldn't be necessary:
```suggestion
python3 -mvenv ./venv --upgrade-deps
./venv/bin/pip3 install transifex-client
echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"api"$'\npassword = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc
./venv/bin/tx push -s
```

Reply:
sorry to bother you @DavidePrincipi but the command above fails, this is the error log traces

```
0.10s$ ./.transifex.sh
We push the translations to transifex
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip]
            ENV_DIR [ENV_DIR ...]
venv: error: unrecognized arguments: --upgrade-deps
./.transifex.sh: line 22: ./venv/bin/pip3: No such file or directory
./.transifex.sh: line 24: ./venv/bin/tx: No such file or directory
```

this is the commit I did https://github.com/NethServer/nethserver-sogo/pull/53

Reply:
Ok, :white_flag: